### PR TITLE
setup.py, README: Generate proper Project Description for PyPi

### PR DESCRIPTION
**Pull request by ix5**
### setup.py: Include README in pypi upload

This allows for more information by embedding the README at https://pypi.org/project/isso/ as a "Project Description".

### README: ~~Point LICENSE to current GH master,~~ shuffle sections a bit

~~Reasoning: Use absolute link to allow embedding the rendered README on pypi (where the relative-path LICENSE is not available).~~

### CODE REVIEW

**Review by jelmer — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Python and JavaScript. It aims to be a drop-in replacement for
 See [posativ.org/isso](http://posativ.org/isso/) for more details.
 
 ## License
-MIT, see [LICENSE](LICENSE).
+MIT, see [LICENSE](https://github.com/posativ/isso/blob/master/LICENSE).
```
Comment (jelmer) [inline README.md:13]: This breaks things for forks, can we just refer to the local file?

**Review by jelmer — CHANGES_REQUESTED** [review]
See below.

File: README.md (hunk 1 above)
Comment (ix5) [reply README.md:13]: You cannot link to package resources inside PyPi, even if you include them in `MANIFEST.in`.
(I tested via https://test.pypi.org/project/isso-testpypi/ and the link goes to a 404, as you'd expect: https://test.pypi.org/project/isso-testpypi/LICENSE )

**Comment by jelmer** [comment]
On Wed, Feb 23, 2022 at 01:45:26PM -0800, ix5 wrote:
> @ix5 commented on this pull request.
> > @@ -9,7 +9,7 @@ Python and JavaScript. It aims to be a drop-in replacement for
>  See [posativ.org/isso](http://posativ.org/isso/) for more details.
>  
>  ## License
> -MIT, see [LICENSE](LICENSE).
> +MIT, see [LICENSE](https://github.com/posativ/isso/blob/master/LICENSE).
> 
> You cannot link to package resources inside PyPi, even if you include them in `MANIFEST.in`.
> (I tested via https://test.pypi.org/project/isso-testpypi/ and the link goes to a 404, as you'd expect: https://test.pypi.org/project/isso-testpypi/LICENSE )

I don't think we should bother fixing it then; ideally we'd just filter out the entire license section since the license is already shown on the left hand side on Pypi.

**Comment by ix5** [comment]
> ideally we'd just filter out the entire license section

Done.

Personally, I think this is the ugliest code I've ever committed, but I agree with your sentiment. Unless someone comes up with a better/cleaner/more beautiful solution, I'm merging this within 2 days.

**Pull request merged**